packages/ember-views/lib/mixins/view_support.js

```diff
@@ -620,7 +620,7 @@ export default Mixin.create({
     this._super(...arguments);
 
     if (!this.elementId && this.tagName !== '') {
-      this.elementId = guidFor(this);
```

Comment:
Figured out the problem. I didn't realize my alias code was getting run on update as well as creation. Seems to work fine now.